### ISSUE TITLE
Allow disabling comment

### ISSUE THREAD

**Issue by shimonelbaz** [mtfoley/pr-compliance-action#327]
**Is your feature request related to a problem? Please describe.**
it would be great to have a boolean flag to choose whether to post a comment or not (specifically in the body check test)
that way if i am using the action with multiple variations i can control when and what to comment

**Describe the solution you'd like**
postComment: true / false

**Describe alternatives you've considered**
overriding the value with "" but it simply posts the default value

**Additional context**
N/A

**Comment by mtfoley** [comment]
Does this mean that you'd want to have failing checks with no guiding comments about what the contributor ought to adjust?

**Comment by shimonelbaz** [comment]
Yes, because i want to perform some more steps anc checks and provide a different comment or not accordingly
